airflow/providers/amazon/provider.yaml

```diff
@@ -455,7 +461,8 @@ hook-class-names:  # deprecated - to be removed after providers add dependency o
   - airflow.providers.amazon.aws.hooks.s3.S3Hook
   - airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook
   - airflow.providers.amazon.aws.hooks.emr.EmrHook
-  - airflow.providers.amazon.aws.hooks.redshift.RedshiftSQLHook
+  - airflow.providers.amazon.aws.hooks.redshift_sql.RedshiftSQLHook
+  - airflow.providers.amazon.aws.hooks.redshift_cluster.RedshiftHook
```

Comment:
@dstandish  - this is the error: by declaring the RedshiftHook as "registered" connection it is registered with the same "connection type" as AwsBaseHook. It was not registered before, so you did not have this error.

RedShiftHook does not provide it's own connection, it piggibacks on existing AwsBaseHook connection so you should not register it here.

Reply:
And Also - hook-class-names is deprecated. It is replaced by `connection-types' below (and it only there to support Airflow 2.1.0).  The "connection types" below are correct - they do not contain the RedshiftHook

Reply:
Sorry for jumping to conclusion @dstandish  -> see the explanation above. I hope it is clear :D

Reply:
ok cool thank you.  i'll "unregister" RedshiftHook